xbmc/utils/ActorProtocol.h

```diff
@@ -51,7 +51,9 @@ class Message
   bool Reply(int sig, void *data = NULL, int size = 0);
 
 private:
-  Message() {isSync = false; data = NULL; event = NULL; replyMessage = NULL;};
+  Message()
+  : signal(0), isSync(false), isSyncFini(false), isOut(false), isSyncTimeout(false),
```

Comment:
you missed the point. members are dependent on state and the value of. i.e. isOut is meaningless when the object gets constructed. This is important information if you want to understand the behaviour. initializing all those members is meaningless to the behaviour and just confuses the reader if not stated explicitly. Some may think it is important that those values are initialized with those values that were taken arbitrarily.